<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AIi2iNodeWidget</class>
 <widget class="QWidget" name="AIi2iNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255,0);
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 3px solid rgb(3, 23, 3);
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
	stop:0 rgb(150,200,170),
	stop:0.167 rgb(150,200,170),
	stop:0.1670001 rgb(240,255,246)
	stop:1 rgb(240, 255, 246));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);
border: none;
font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>图生图</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="showLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>126</y>
      <width>258</width>
      <height>158</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,0);
border-color: rgba(255, 255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>此处显示图像</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-radius: 10px;
image: url(:/nodewidget/AIP2P.png);
background: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="viewCheckBox">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>90</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border: none;
background-color: rgba(255, 255, 255, 0);
font: 10pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>120</y>
      <width>270</width>
      <height>170</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px dashed rgb(100, 100, 100);
border-radius: 10px;
background-color:rgb(230, 230, 230);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="beginButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    image: url(:/nodewidget/begin.png);
	border-radius: 5px;
	border: 0px;
	background-color: rgb(219,241,227);
}

QPushButton:hover {
	background-color: rgb(150,200,170);
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid rgb(150,200,170);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>55</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    image: url(:/nodewidget/edit.png);
	border-radius: 5px;
	border: 0px;
	background-color: rgb(219,241,227);
}

QPushButton:hover {
	background-color: rgb(150,200,170);
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid rgb(150,200,170);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="viewButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>55</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    image: url(:/nodewidget/check.png);
	border-radius: 5px;
	border: 0px;
	background-color: rgb(219,241,227);
}

QPushButton:hover {
	background-color: rgb(150,200,170);
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid rgb(150,200,170);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="promptLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>190</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid rgb(180,180,180);
background-color: rgb(228,240,229);
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>nameLabel</zorder>
   <zorder>showLabel</zorder>
   <zorder>label_3</zorder>
   <zorder>viewCheckBox</zorder>
   <zorder>beginButton</zorder>
   <zorder>editButton</zorder>
   <zorder>viewButton</zorder>
   <zorder>promptLabel</zorder>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	border-radius: 10px;
	border-image: url(:/nodewidget/close.png);
	background: rgb(255,255,255);
}

QPushButton:hover {
	background-color: rgb(220,220,220);
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid rgb(220,220,220);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
